### PR TITLE
chore: remove duplication in parameter definitions

### PR DESCRIPTION
**Pull request by OrKoN**
Technically parameters for each of the commands are different but for the user convenience most of the should work the same way so we can re-use their definitions.

### CODE REVIEW

File: packages/browsers/src/CLI.ts [hunk 1]
```diff
@@ -50,33 +50,86 @@ export class CLI {
     this.#cachePath = cachePath;
   }
 
+  #defineBrowserParameter(yargs: yargs.Argv<unknown>) {
```
Comment (Lightning00Blade) [inline packages/browsers/src/CLI.ts]: Nit: return type

File: packages/browsers/src/CLI.ts [hunk 2]
```diff
@@ -50,33 +50,86 @@ export class CLI {
...
+              browser: args.browser.name,
+              buildId: args.browser.buildId,
+              cacheDir: args.path ?? this.#cachePath,
+              platform: args.platform,
+            })}`
+          );
+        }
+      )
+      .command(
+        'downloads <browser>',
```
Comment (Lightning00Blade) [inline packages/browsers/src/CLI.ts]: Is the idea for this to be an alias to `install`, the code looks the same? Yargs supports that -> https://github.com/yargs/yargs/blob/main/docs/advanced.md#command-aliases

Comment (OrKoN) [reply packages/browsers/src/CLI.ts]: I have forgotten to remove the command I have been trying out locally.

File: packages/browsers/src/CLI.ts (hunk 1 above)
Comment (OrKoN) [reply packages/browsers/src/CLI.ts]: Done

**Review by Lightning00Blade — APPROVED** [review]

**Pull request merged**